include/PR/gbi.h

```diff
@@ -1277,15 +1285,16 @@ typedef struct {
   char 		pad3;
 } Light_t;
 
-// Added in MM
+#ifdef F3DEX_GBI_PL
 typedef struct {
-  unsigned char col[3];
-  unsigned char kc;
-  unsigned char colc[3];
-  unsigned char kl;
-  short pos[3];
-  unsigned char kq;
+  unsigned char col[3];		/* point light value (rgba) */
+  unsigned char kc;         /* constant attenuation (> 0 indicates point light) */
+  unsigned char colc[3];	/* copy of point light value (rgba) */
+  unsigned char kl;         /* linear attenuation */
+  short pos[3];             /* world-space position of light */
+  unsigned char kq;         /* quadratic attenuation */
```

Comment:
tabs :despair:

Reply:
The entire file needs reformatting so I didn't bother with it yet. I plan to follow-up with a formatting pass on gbi.h that makes it the same as OoT.